webseer_process.php

```diff
@@ -185,7 +185,7 @@
 		if ($results['result'] == 0) {
 			$url['failures'] = $url['failures'] + 1;
 //			//////////if ($url['failures'] > $url['downtrigger'] && $url['triggered'] == 0) {
-			if ($url['failures'] >= $tf && $url['triggered'] == 0) {
+			if ($url['failures'] >= ($url['downtrigger'] * 60)/$poller_interval && $url['triggered'] == 0) {
```

Comment:
Not sure that this right.  If you consider that downtrigger is 2 (down for 2 polls), then 2*60 = 120 / 300 (5 minute poller) seems wrong.

Reply:
@cigamit may be able to (dis)agree with me on that.

Reply:
Perhaps I'm misinterpreting the downtrigger value? The description in the web interface says "How many minutes the URL must be down", not how many polling cycles to be down. In that case if downtrigger is 2 minutes in a 5 minute polling cycle you should only need one failure to trigger (which is >= 2*60/300)...

Reply:
OK, I'll roll with that.